### ISSUE TITLE
Any chance of getting it working 2016 

### ISSUE THREAD

**Issue by suedi** [samuellb/fribid#13]
Have linux on all machines and my bank now forces me to use bankid.

Having been windows free for many years it irks me to have to go back to it again.

Maybe I should ditch my f***ing bank instead?

**Comment by Olangu** [comment]
You can use http://www.android-x86.org/ in a virtual machine and install mobile bankid on it.
I think that's your best bet to stay windows free.

**Comment by samuellb** [comment]
There haven't been any progress on this lately. In the mean time you can use the Android version. It's apparently possible to run in inside Chrome:

https://forum.fribid.se/viewtopic.php?id=305
https://wiki.ubuntu.com/SwedishTeam/Support/BankID_i_Linux_genom_Google_Chrome

**Comment by suedi** [comment]
But only with mobile bankid? not cardreader type?

Thanks for links will check out.

May have to run win in virtual machine then if I wanna use cardreader.

I'm possibly gonna try out the chrome version. would like it on a usb so I can totally disconnect
from a computer though. Could you place key on USB?

**Comment by suedi** [comment]
I experimented and got chrome and bankid extension running.

Put them in squashfile and use them in an overlay filesystem on a linux live cd.

The app is running I however have not applied for a bankid with my bank so
will get that and test out.

My goal is to have everything on a USB stick. Chrome + ARC +  bankid extension + bankid key
and maybe I could have this encrypted for safety

**Comment by samuellb** [comment]
Yes this is only mobile bankid. I think it would be hard to separate the keys in a secure way, but I guess you could put your chrome directory on a USB stick and then mount it manually before starting chrome.

What I do personally is that I use an old Android phone with a broken battery as a dedicated BankID device. That's both very simple and secure.

**Comment by suedi** [comment]
When I tried to apply for bankid they wanted me to put in mobilephone number.

This shouldn't be required right? you could have like a android tablet via wifi and bankid and
it should work.

Could you enlighten me on this, please?

I don't want to give up more info than I have to

**Comment by samuellb** [comment]
I think I was able to get a Mobile BankID from Swedbank to a phone without a SIM card. But it might work differently for different banks. Or maybe the phone number is just for contacting you?

**Comment by suedi** [comment]
OK got it working now.

A bit of a hassle though

tried to do a cp -a -r but got something like a permission denied  and then my 
session was terminated? 

Is bankid that aggressive? 

Lost one bankid this way, Since I couldn't get it back I guess no one else can either, right?

Anyway found a way around it, now I can load bankid from a USB key
and use it exactly only when I want to.

DId it your way: "`put your chrome directory on a USB stick and then mount it manually before starting chrome.`"

No need for sim card, they just want all your information in the most Orwell'an way.

Thanks for all your support

closing...

**Comment by samuellb** [comment]
You're welcome. Nice that it worked :)

**Comment by suedi** [comment]
Now they have taken away the possibility to use chrome and bankid.

I am not sure what to do 

Should I 

1) Run windows in a virtual machine and use that
2) Run android-x86.org a virtual machine and use that however since it cannot use app store 
     I am not sure how to get bankid app on there
3) Use an old Android phone with a broken battery as a dedicated BankID device.

Could you expand on how you 're setup is regarding 3.
How do you use it?

**Comment by samuellb** [comment]
Oh, that's bad. It also seems that they have dropped support for Android < 4.1 so phones older than a couple of years wont work either.

However, with option 2 you can download the app outside of the app store.

It could also be worth testing the most recent version of Wine. Who knows, it might be able to run BankID know.
